www/src/providers/Auth0ProviderWithHistory.tsx

```diff
@@ -17,7 +17,7 @@ const Auth0ProviderWithHistory: FC<Auth0ProviderWithHistoryProps> = ({ children
     };
 
     return (
-        <Auth0Provider domain={domain} clientId={clientId} redirectUri={window.location.origin} onRedirectCallback={onRedirectCallback} audience={config.server.audience} scope={'call:ping'}>
```

Comment:
LMAO, love that this was still there.

Reply:
Watch me slip `call:ping` to bypass authorization and authentication 😂